### PR TITLE
Edited fix and resolve endpoints to support comments

### PR DESCRIPTION
**Pull request by mgpaja8**

### CODE REVIEW

**Comment by mgpaja8** [comment]
I dont know why it doesnt give me option to connect this pr to issue..... Sorry Jon :(

**Comment by j0nscalet** [comment]
@mgpaja8  No worries man, we'll figure it out. It may be a permissions thing.

File: src/QualityKitchen.Application/ApplicationServices/IncidentApplicationService.cs [hunk 1]
```diff
@@ -125,19 +125,19 @@ public ServiceResult ResolveIncident(Guid incidentId, Guid managerId, string fin
 
                 Employee manager = operation.FindEmployee(managerId);
                 
-                incident.Resolve(manager.Id, manager.FullName, manager.IsManager, finalComment);
+                Activity resolveIncident = incident.Resolve(manager.Id, manager.FullName, manager.IsManager, finalComment);
```
Comment (j0nscalet) [inline src/QualityKitchen.Application/ApplicationServices/IncidentApplicationService.cs]: Variable name should be `resolvedIncident`

File: src/QualityKitchen.Application/ApplicationServices/IncidentApplicationService.cs [hunk 2]
```diff
@@ -125,19 +125,19 @@ public ServiceResult ResolveIncident(Guid incidentId, Guid managerId, string fin
 
                 Employee manager = operation.FindEmployee(managerId);
                 
-                incident.Resolve(manager.Id, manager.FullName, manager.IsManager, finalComment);
+                Activity resolveIncident = incident.Resolve(manager.Id, manager.FullName, manager.IsManager, finalComment);
 
                 this._incidentRepository.Store(incident);
 
-                return ServiceResult.Success();
+                return ServiceResult.Success<Activity>(resolveIncident);
```
Comment (j0nscalet) [inline src/QualityKitchen.Application/ApplicationServices/IncidentApplicationService.cs]: See above

**Comment by mgpaja8** [comment]
sure, im still at work :)
thanks

**Comment by j0nscalet** [comment]
@mgpaja8 created #33 out to this.

**Pull request merged**